deps.bzl

```diff
@@ -2394,8 +2442,12 @@ def install_go_mod_dependencies(workspace_name = "buildbuddy"):
     go_repository(
         name = "com_github_hanwen_go_fuse_v2",
         importpath = "github.com/hanwen/go-fuse/v2",
-        sum = "h1:0Aex0XJ12fMbuuwyowIbUKBs976RxRuBxlYVK3eoHTA=",
-        version = "v2.5.2-0.20240816120607-db8da281991c",
+        patch_args = ["-p1"],
+        # TODO: Remove when/if hanwen/go-fuse#525 is merged. Prevents a kernel
+        # panic during init if /dev/null does not exist.
+        patches = ["@{}//buildpatches:com_github_hanwen_go_fuse_v2.patch".format(workspace_name)],
+        sum = "h1:OQBE8zVemSocRxA4OaFJbjJ5hlpCmIWbGr7r0M4uoQQ=",
+        version = "v2.5.1",
```

Comment:
We already upgraded this to the exact commit with the fix.

Reply:
You're right; reverted.